cypress/tests/ImageElement.cy.ts

```diff
@@ -597,19 +597,6 @@ describe("ImageElement", () => {
           })
         );
       });
-      it("should add nested repeater elements", () => {
-        addImageElement();
-        clickButton(AddRepeaterButtonId);
-        clickButton(AddNestedRepeaterButtonId);
-        getElementRichTextField("nestedRepeaterText").should("exist");
-      });
-      it("should remove nested repeater elements", () => {
-        addImageElement();
-        clickButton(AddRepeaterButtonId);
-        clickButton(AddNestedRepeaterButtonId);
-        clickButton(RemoveNestedRepeaterButtonId);
-        getElementRichTextField("nestedRepeaterText").should("not.exist");
-      });
```

Comment:
These tests are now covered in `RepeaterFieldView.spec.ts` - we could have both unit and integration tests but this seems like overkill.

Reply:
I've re-instated these tests (but fixed) in https://github.com/guardian/prosemirror-elements/pull/334/commits/5c48fed79d9548b2f50e55fc617d5c4169224408, as these integration tests more closely test the experience of the user and the unit tests test the building blocks - a worthy distinction IMO.

https://github.com/guardian/prosemirror-elements/pull/334/commits/5c48fed79d9548b2f50e55fc617d5c4169224408 also fixes the outstanding test that was broken in https://github.com/guardian/prosemirror-elements/pull/334/commits/de60847f0e1fe9417134f8019cea148f1959328f.

Reply:
The fixes mentioned above relate to handling the fact we no longer allow removal of repeater children when there's only one. @fredex42 can I confirm this is desired behaviour for all uses of the repeater in recipes? If not, then we'll need to make `minChildren` a prop of repeater to support both use cases (and the tests will need to be updated accordingly).

Reply:
I think we'll want to support repeater fields that have no children in general, though the Key Takeaways will always want at least one child.

Reply:
Interesting - would a repeater field have any functionality without a child though?

Reply:
I'm struggling to think of a useful example, but I can imagine a field where we might want to support 0-many of something.

Reply:
@Fweddi @rhystmills @twrichards I am afraid that in recipes we definitely do need to support 0 repeater children; this is to handle the case where the structuriser fails to find something, e.g. serving quantities or timings.  The form in that case needs to show 0 of that thing and allow the user to then add it manually (as there is no sensible default that can be applied in that case)